### PR TITLE
Sept updates keep on coming...

### PR DESCRIPTION
**Pull request by froboy**
- https://yusa.atlassian.net/browse/DS-1072 How to > Schema 
- https://yusa.atlassian.net/browse/DS-1050 User > Content Types > Layout Builder Landing Page
- https://yusa.atlassian.net/browse/DS-820 How to > Layout Builder migration
- https://yusa.atlassian.net/browse/DS-811 User > Layout Builder > Advanced(/Global Styles)
- UI Kit WIP at /ui-kit (unlinked)

### CODE REVIEW

File: content/en/docs/howto/migrate-to-lb/_index.md [hunk 1]
```diff
@@ -1,16 +1,137 @@
...
+: [Cards](../../user-documentation/layout-builder/cards) or [Grid CTA](../../user-documentation/layout-builder/grid-cta)
+
+Gallery
+: [Carousel](../../user-documentation/layout-builder/carousel)
+
+Grid Content
+: [Grid CTA](../../user-documentation/layout-builder/grid-cta) or [Icon Grid](../../user-documentation/layout-builder/icon-grid)
+
+Limited Time Offer
+: None (or Alert?)
```
Comment (podarok) [inline content/en/docs/howto/migrate-to-lb/_index.md:105]: Banner? @froboy 
Should be banner

File: content/en/docs/howto/use-structured-data/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+If an Accordion section contains Frequently Asked Questions, check the **FAQ?** checkbox to output them as structured data.
+
+Tips for writing good FAQ content:
+
+- Ensure the content contains individual sets of questions ("How old does my child need to be to swim at the YMCA?") and answers ("The YMCA offers swim classes starting at age 3 and the pool is open to children of all ages with parental supervision").
+- Only one FAQ should be added to a page. If more than one is added, only the first will be output to the structured data.
+
+### Customizing FAQs
+
+Due to the complexity of the FAQ data, the structured data is [managed entirely in code](https://git.drupalcode.org/project/lb_accordion/-/blob/1.0.x/lb_accordion.module?ref_type=heads#L40) and is not customizable via the Drupal admin.
```
Comment (podarok) [inline content/en/docs/howto/use-structured-data/_index.md:52]:
```suggestion
Due to the complexity of the FAQ data, the structured data is [managed entirely in code](https://git.drupalcode.org/project/lb_accordion/-/blob/1.0.x/lb_accordion.module?ref_type=heads#L40) and is not customizable via the Drupal admin. If you need any specific customisation, please post your ideas in Slack or suggest to online Roadmap for the core team to discuss and plan implementation
```

File: content/en/docs/user-documentation/layout-builder/advanced-options/_index.md [hunk 3]
```diff
@@ -6,13 +6,40 @@ weight: -5
...
+{{< youtube S9QninI7ydM >}}
+
 ## Y Styles
 
-These options provide customizations of Layout Builder-enabled pages at the Content Type, Page, and Component level.
+These options provide customizations of Layout Builder-enabled pages at the Content Type, Page, and Component(/Block) level.
+
+Styles inherit from content types, to pages, to components. Some styles can also be overridden at each level - block styles can override page styles, which can override content type styles.
+
+```mermaid
```
Comment (podarok) [inline content/en/docs/user-documentation/layout-builder/advanced-options/_index.md:17]: cool

File: content/en/docs/user-documentation/layout-builder/advanced-options/lb-advanced--banner-variants.png
Comment (podarok) [inline content/en/docs/user-documentation/layout-builder/advanced-options/lb-advanced--banner-variants.png:1]: Item 5 here is a Limited Time Offer @froboy

**Review by podarok — APPROVED** [review]
Small things @froboy

File: content/en/docs/howto/use-structured-data/_index.md (hunk 2 above)
Comment (froboy) [reply content/en/docs/howto/use-structured-data/_index.md:52]: @podarok nice. Just a couple grammar tweaks.

```suggestion
Due to the complexity of the FAQ data, the structured data is [managed entirely in code](https://git.drupalcode.org/project/lb_accordion/-/blob/1.0.x/lb_accordion.module?ref_type=heads#L40) and is not customizable via the Drupal admin. If you need specific customizations, please post your ideas in Slack or suggest them on [the Roadmap](/roadmap) for the core team to discuss and implement.
```

File: content/en/docs/user-documentation/layout-builder/advanced-options/_index.md (hunk 3 above)
Comment (froboy) [reply content/en/docs/user-documentation/layout-builder/advanced-options/_index.md:17]: 🧜‍♀️ there's support for a ton more too: https://www.docsy.dev/docs/adding-content/diagrams-and-formulae/

**Pull request merged**